openreview/venue_request/process/revisionProcess.py

```diff
@@ -53,11 +53,15 @@ def process(client, note, invitation):
                     'No, I don\'t want to reveal any author identities.'],
                 'required': True
             }
-
+        decision_options = forum_note.content.get('decision_options')
+        if decision_options:
+            decision_options = [s.translate(str.maketrans('', '', '"\'')).strip() for s in decision_options.split(',')]
+        else:
+            decision_options = ['Accept (Oral)', 'Accept (Poster)', 'Reject']
```

Comment:
Can you add a test here: https://github.com/openreview/openreview-py/blob/fix/homepage_tabs/tests/test_venue_request.py#L861

and verify the Post_Decision_Stage invitation default value is correctly computed?